backend/libraries/ballerina-core/codegen/python/language constructs/record.fs

```diff
@@ -0,0 +1,52 @@
+namespace Ballerina.DSL.Codegen.Python.LanguageConstructs
+
+
+module Record =
+
+  open Ballerina.Core.StringBuilder
+  open Ballerina.DSL.Codegen.Python.LanguageConstructs.Model
+
+  let private indent = (+) "    "
```

Comment:
this should be defined once for the entire python codegen module

Reply:
Removed duplication